release/NativeEditPlugin/scripts/MaskOptions.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+
+/// <summary>
+/// Please see <see href="https://github.com/RedMadRobot/input-mask-android/wiki"/> for more information about
+/// the masks.
+/// </summary>
+public class MaskOptions
+{
+	public enum AffinityCalculationStrategy
```

Comment:
Please, add a comment pointing to the method in the java file that does the casting